langchain/text_splitter.py

```diff
@@ -34,6 +34,23 @@
 TS = TypeVar("TS", bound="TextSplitter")
 
 
+def _make_spacy_pipeline_for_splitting(pipeline: str) -> Any:  # avoid importing spacy
+    try:
+        import spacy
+    except ImportError:
+        raise ImportError(
+            "Spacy is not installed, please install it with `pip install spacy`."
+        )
+    if pipeline == "sentencizer":
+        from spacy.lang.en import English
+
+        sentencizer = English()
+        sentencizer.add_pipe("sentencizer")
+    else:
+        sentencizer = spacy.load(pipeline, disable=["ner"])
```

Comment:
As a small tip, you can use `exclude` instead of `disable` if you don't ever plan on using the `ner` component. By excluding it, it isn't even loaded and won't be serialized. If you only disable it, it'll be loaded and serialized, though not run until you enable it again.

Reply:
Also, do you need the `tagger`? If not, you can exclude/disable that, too.

Reply:
Eh I just opened it, it's not even a line.